pages/deposit.tsx

```diff
@@ -86,7 +86,7 @@ const UsdtIcon = (props: SVGProps<SVGSVGElement>) => (
 );
 
 const DepositMethodItems = ["buy", "deposit"] as const;
-type DepositMethod = typeof DepositMethodItems[number];
+type DepositMethod = (typeof DepositMethodItems)[number];
```

Comment:
Is this purely a formatting change?

Reply:
yes, I think prettier automatically applied this